### PR TITLE
Dateline spacing

### LINKED ISSUE #5269: Bug: Container dateline positioned wrong at tablet
Image

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
This PR fixes #5269 by refactoring and adjusting the spacing shown around the dateline, which appears on certain containers

## Why?
There was [an issue](https://github.com/guardian/dotcom-rendering/issues/5269) previously where spacing broke at `tablet` and in general was not completely in line with Frontend. @HarryFischer what do you think of space showing below the dateline now when below `tablet`?

| Before      | After      |
|-------------|------------|
|  |  |

[before]: https://example.com/before.png
[after]: https://example.com/after.png

### CODE REVIEW

**Comment by HarryFischer** [comment]
Looks good @oliverlloyd

**Review by ioannakok — APPROVED** [review]
LGTM!

File: dotcom-rendering/src/web/components/ContainerTitle.tsx [hunk 1]
```diff
@@ -60,13 +50,22 @@ const descriptionStyles = (fontColour?: string) => css`
...
+	margin-left: 0;
 	${between.tablet.and.leftCol} {
 		margin-left: 10px;
 	}
 
-	${until.leftCol} {
-		margin-bottom: ${space[4]}px;
+	${from.leftCol} {
+		margin-left: 0;
 	}
```
Comment (mxdvl) [inline dotcom-rendering/src/web/components/ContainerTitle.tsx:67]: Is this not redundant? Assuming the `between.tablet.and.leftCol` no longer applies

Comment (oliverlloyd) [reply dotcom-rendering/src/web/components/ContainerTitle.tsx]: You're right. Removed

**Comment by mxdvl** [comment]
Hey there “external contributor”! If you are unable to merge this PR, just let us know and we’ll get this sorted for you 😉

**Comment by mxdvl** [comment]
Just to make sure this branch does not fall out of sync with `main`, I will merge this in!

**Comment by oliverlloyd** [comment]
> Hey there “external contributor”! If you are unable to merge this PR, just let us know and we’ll get this sorted for you 😉

Oh, hello there! Sorry, just saw this. I was on holidays walking Maggie in the hills.

Thanks for merging!

**Pull request merged**